R/html_document.R

```diff
@@ -287,6 +284,24 @@
 #'  to via mouse clicks.} \item{\code{print} (defaults to \code{TRUE}) controls
 #'  whether the table of contents appears when user prints out the HTML page.}}
 #'
+#'@section Code folding:
```

Comment:
I don't know why we omitted the spaces before all `@section`  tags in this file, but I'd like to add them back.

```suggestion
#' @section Code folding:
```